CHANGELOG.txt

```diff
@@ -121,8 +127,14 @@ Improvements
 
 Bug Fixes
 """""""""
+- Make REST API patch user endpoint work the same way as the UI (#18757)
+- Don't ignore legacy ``concurrency`` dag parameter (#18730)
```

Comment:
```suggestion
```

This can go with

https://github.com/apache/airflow/blob/25cfb71ade9b78d37b3d73afe0e97cf54991f24f/CHANGELOG.txt#L124